src/utils/DEMUtils.js

```diff
@@ -169,7 +169,7 @@ function tileAt(pt, tile) {
                 return t;
             }
         }
-        if (tile.getLayerTextures(l_ELEVATION)[0].coords.zoom > -1) {
+        if (tile.material.textures[l_ELEVATION][0].coords.zoom > -1) {
```

Comment:
Is it equivalent to `isElevationLayerLoaded` ? (which would be clearer)

Reply:
Yes.

I'll use this *but* I'll remove it from TileMesh and move it where it belongs = LayeredMaterial.